src/main/kotlin/no/nav/syfo/infrastructure/database/DialogmotekandidatVurderingRepository.kt

```diff
@@ -35,6 +36,30 @@ class DialogmotekandidatVurderingRepository(private val database: DatabaseInterf
         if (commit) connection.commit()
     }
 
+    override suspend fun createUnntak(connection: Connection, unntak: Unntak) {
```

Comment:
Ser litt rart ut med en repository-tjeneste som tar inn en `connection`. Men vi må kanskje det for å koordinere noen greier?

Reply:
Yup 🥲 Men unngår det om vi gjør som du foreslår over her 🙌

Reply:
Tabellen `dialogmotestatus` tror jeg bare er dupliserte data fra `isdialogmote` (lest inn fra Kafka). Så det er `dialogmotekandidat_endring` som er tabellen som forteller hvordan kandidatstatus har endret seg over tid for en gitt person.